awx/main/scheduler/task_manager_models.py

```diff
@@ -277,7 +277,6 @@ def init_with_consumed_capacity(cls, **kwargs):
             from awx.main.models import UnifiedJob
 
             if instance_group_queryset is not None:
-                logger.debug("******************INSTANCE GROUP QUERYSET PASSED -- FILTERING TASKS ****************************")
```

Comment:
@TheRealHaoLiu this is a log that should have stood out to us, but I don't think it ever did because it was never configured like it should have been.